### PR TITLE
Add maketarget for serviceaccount-secret-rotation

### PR DESCRIPTION
**Pull request by alvaroaleman**
Ref https://issues.redhat.com/browse/DPTP-1897

/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -274,7 +274,10 @@ secrets:
 	hack/secrets.sh $(kerberos_id) $(kubeconfig_path) $(bw_password_path)
 .PHONY: secrets
 
+serviceaccount-secret-rotation:
+	go get github.com/openshift/ci-tools/cmd/serviceaccount-secret-rotation-trigger
```
Comment (hongkailiu) [inline Makefile:278]: Do we want to use the image?
https://github.com/openshift/release/blob/2c5aa9c51abb17f55d9d3d8629f57fa3d7110d28/ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml#L410

File: Makefile [hunk 2]
```diff
@@ -274,7 +274,10 @@ secrets:
...
+	time serviceaccount-secret-rotation-trigger --dry-run=false  --namespace=ci 2>&1|tee /tmp/out.json
```
Comment (petr-muller) [inline Makefile:279]: Just curious, why the `time` and the `tee`? Debugging leftovers?

Comment (alvaroaleman) [reply Makefile]: Yeah, updated the PR to instead just create a periodic

**Review by petr-muller — APPROVED** [review]

File: ci-operator/jobs/infra-periodics.yaml [hunk 3]
```diff
@@ -59,6 +59,48 @@ periodics:
...
+  cron: '@yearly'
+  decorate: true
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-rotate-serviceaccount-secrets
+  spec:
+    containers:
+    - args:
+      - --namespace=ci
+      - --dry-run=true
```
Comment (petr-muller) [inline ci-operator/jobs/infra-periodics.yaml:73]: I assume this will be flipped afterwards.

Comment (alvaroaleman) [reply ci-operator/jobs/infra-periodics.yaml:73]: Yeah I mostly left it so we don't get surprised once in a year :upside_down_face:

**Pull request merged**